modules/scheduling-a-pipeline-run.adoc

```diff
@@ -18,18 +18,18 @@ endif::[]
 * You have imported a pipeline to an active pipeline server.
 
 .Procedure
-. From the {productname-short} dashboard, click *Data Science Pipelines* -> *Runs*.
-+
-The *Runs* page opens.
-. From the *Project* list, select the project that you want to schedule a run for.
+. From the {productname-short} dashboard, click *Experiments* -> *Experiments and runs*.
+. On the *Experiments* page, select the *project* that contains the pipeline experiment for the run that you want to schedule.
+. From the list of pipeline experiments, click the pipeline experiment that contains the run that you want to schedule. 
 . Click the *Schedules* tab.
-. Click *Schedule run*. 
+. Click *Create schdule*. 
 . On the *Schedule run* page, configure the run that you are scheduling:
+.. From the *Experiment* list, select the pipeline experiment to contain the pipeline run that you want to schedule. Alternatively, to create a new pipeline experiment, click *Create new experiment* and complete the relevant fields in the *Create experiment* dialog.
```

Comment:
- Suggest updating to:
 From the *Experiment* list, select the pipeline experiment that you want to contain the scheduled run.

- Suggest updating to use ', and then' construction:
'*Create new experiment*, and  then complete'

Reply:
Changed to your suggestions.